README.md

```diff
@@ -56,35 +57,47 @@ namespace rootfinding {
 
 The implementation would look like
 
-```{.cpp #algorithm}
-// this C++ code snippet is later referred to as <<algorithm>>
-#include "newtonraphson.hpp"
+```{.hpp file=src/algebra.hpp}
+// this C++ code snippet is later referred to as <<algebra>>
 
-namespace rootfinding
+namespace algebra
 {
 
-// An example function is x^3 - x^2  + 2
-double func(double x)
+// An example equation is x^3 - x^2  + 2
+double equation(double x)
 {
   return x * x * x - x * x + 2;
 }
 
-// Derivative of the above function which is 3*x*x - 2*x
-double derivFunc(double x)
+// Derivative of the above equation which is 3*x^x - 2*x
```

Comment:
`x^x` is a regression, it should be `x*x`, please merge master into this branch.

Reply:
done